org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/workspace/WorkbenchShutdownEvent.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016 the original author or authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.buildship.core.workspace;
+
+import org.eclipse.buildship.core.event.Event;
+
+/**
+ * Project shutdown event.
```

Comment:
Let's not repeat the obvious in comments, as that's already captured by the class name. Also note the copy-paste error :)

Instead the javadoc should say when this event occurs (i.e. probably before the workbench is shutdown, the listener can react synchronously I guess?)